src/main/java/org/spongepowered/api/world/WorldBorder.java

```diff
@@ -194,4 +201,109 @@
      * @param damage The damage amount
      */
     void setDamageAmount(double damage);
+
+    /**
+     * Returns a new builder for creating a task to pre-generate the chunks
+     * inside the border. The current state of the border is "snapshoted" for
+     * the builder. It is not backed by this instance. If the border size is
+     * changing then the target size is used.
```

Comment:
The target is by definition the new size. The "target" as in "the size that is going to be reached".